docs/pages/database-access/troubleshooting.mdx

```diff
@@ -139,16 +139,14 @@ For more detailed information about database access controls and how to restrict
 access see the [RBAC](../database-access/rbac.mdx) documentation.
 
 ## Connection to MySQL database results in "Unknown system variable 'query_cache_size'" error
-When a Teleport Proxy is configured with Teleport Proxy Listener Mode (`proxy_listener_mode: separate`) by default MySQL client connections to the Teleport Proxy don't use the TLS Routing feature that allows propagating correct MySQL Server Version.
+When TLS Routing feature is disable by default, the Teleport Proxy Service returns `8.0.0-Teleport` as the MySQL server version. In some cases, like connecting with a GUI Client, this can result in obtaining an `Unknown system variable 'query_cache_size'` error that indicates that MySQL capabilities were not properly negotiated between the MySQL client and server.
```

Comment:
```suggestion

When TLS Routing is disable by default, the Teleport Proxy Service returns `8.0.0-Teleport` as the MySQL server version. In some cases, like connecting with a GUI Client, this can result in obtaining an `Unknown system variable 'query_cache_size'` error that indicates that MySQL capabilities were not properly negotiated between the MySQL client and server.
```
- Let's standardize on an empty line between a heading and the first paragraph after it
- Grammar tweak